test/contracts-network/meta-colony.js

```diff
@@ -410,6 +417,60 @@ contract("Meta Colony", (accounts) => {
     });
   });
 
+  describe("when managing global skills", () => {
+    let globalSkillId;
+    beforeEach(async () => {
+      const { OldInterface } = await deployOldColonyVersion(
+        "Colony",
+        "IMetaColony",
+        [
+          // eslint-disable-next-line max-len
+          "Colony,ColonyDomains,ColonyExpenditure,ColonyFunding,ColonyPayment,ColonyRewards,ColonyRoles,ColonyTask,ContractRecovery,ColonyArbitraryTransaction",
+        ],
+        "glwss4",
+        colonyNetwork,
+      );
+
+      await downgradeColony(colonyNetwork, metaColony, "glwss4");
+
+      // Make the colonyNetwork the old version
+      await deployOldColonyNetworkVersion(
+        "",
+        "IColonyNetwork",
+        ["ColonyNetwork,ColonyNetworkAuction,ColonyNetworkDeployer,ColonyNetworkENS,ColonyNetworkExtensions,ColonyNetworkMining,ContractRecovery"],
+        "glwss4",
+      );
+
+      const colonyNetworkAsEtherRouter = await EtherRouter.at(colonyNetwork.address);
+      const latestResolver = await colonyNetworkAsEtherRouter.resolver();
+
+      await downgradeColonyNetwork(colonyNetwork, "glwss4");
+
+      // Add global skill
+      const oldMetaColony = await OldInterface.at(metaColony.address);
+      await oldMetaColony.addGlobalSkill({ from: accounts[0] });
```

Comment:
We don't explicitly use `accounts[0]` elsewhere in the tests, so I don't see why we should start doing so now